modules/ROOT/pages/_partials/howto/how-to-create-users.adoc

```diff
@@ -74,6 +94,44 @@ Create users by hardcoding their credentials in the Configuration Properties fil
 This method is convenient for testing and to get started. +
 Use the Admin REST API for production system changes.
 
+[#{tabs}]
+======
+
+Using named collections::
++
+---
+
+[source,json]
+----
+{
+  "databases": {
+    "mydatabase": {
+      "users": { // <.>
+        "GUEST": {"disabled": true},
+        "Edge1User": {
+          "password": "pass", // <.>
+          "collection_access": {
+            "scopename": {
+              "collectionname": {
+                "admin_channels": ["RandomChannel"]
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+----
+
+<.> {configuration-schema-database--pfx--db}-users[databases.$db.users]
+<.> Here we add the Edge1 user with collection-aware channel access
```

Comment:
```suggestion
<.> Here we add the Edge1 user with access to channel `RandomChannel` in the scope `scopename` and collection `collectionname`.
```

FYI, a channel is scope to a specific scope and collection so you can multiple collections, you could have the same channel assigned to each collection.